tests/test.php

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
What is the meaning of this file?

Reply:
That's a simple php file to test the library manually. I've added it, as it should be handy to have for testing future changes

Reply:
Would it make sense to add a doc block to the beginning of this file stating the purpose and the reasoning behind it?